lib/src/observable.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:async';
+import 'dart:ffi';
+import 'bindings/bindings.dart';
+import 'bindings/signatures.dart';
+
+import 'store.dart';
+import 'query/query.dart';
+
+// ignore_for_file: non_constant_identifier_names
+
+// dart callback signature
+typedef Any = void Function(Pointer<Void>, Pointer<Uint32>, int);
+
+class Observable {
+
+  static final anyObserver = <int, Pointer<Void>>{};
+  static final any = <int, Any>{}; // radix? > tree?
+
+  // sync:true -> ObjectBoxException: 10001 TX is not active anymore: #101
+  static final controller = StreamController<int>.broadcast();
+
+  static void _anyCallback(Pointer<Void> user_data, Pointer<Uint32> mutated_ids, int mutated_count) {
+    for(var i=0; i<mutated_count; i++) {
+      // call schema's callback
+      if (any.containsKey(mutated_ids[i])) {
+        any[mutated_ids[i]](user_data, mutated_ids, mutated_count);
+      }
+    }
+  }
+
+  static void subscribe(Store store) {
+    final callback = Pointer.fromFunction<obx_observer_t<Void, Uint32>>(_anyCallback);
```

Comment:
This does not appear right (Dart error). Shouldn't the second type be `Pointer<Uint32>`? Then `_anyCallback` needs updating as well.
Anyhow, I would recommend avoiding type args if not really necessary when defining the functions in `signatures.dart` (`typedef obx_observer_t` etc.). They make things needlessly complicated.